.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,8 @@
+Before opening 'pull request'
 
+** Search existing issues and pull requests to see if issue was already discussed.
+** Check our discussions to see if issue was already discussed.
+** Check for specific project to raise issue on that we support under [spotbugs](https://github.com/spotbugs)
```

Comment:
"Check for a specific project that we support to raise the issue on, under"

Reply:
done